Formula/povray.rb

```diff
@@ -39,6 +39,7 @@ def install
       --disable-dependency-tracking
       --prefix=#{prefix}
       --mandir=#{man}
+      --with-boost=#{HOMEBREW_PREFIX}
```

Comment:
```
      --with-boost=#{Formula["boost"].opt_prefix}
```

Reply:
done, and changed `--with-openexr` as well for consistency.